src/style.css

```diff
@@ -1,3 +1,86 @@
+:root {
+  --main-color: rgba(0, 172, 220, 1);
+  --header-height: 60px;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+ul,
+li,
+a {
+  all: unset;
+}
+
+h1 {
+  display: none;
```

Comment:
Hiding the `<h1>` element with `display: none;` might affect accessibility, as screen readers use headings to navigate the page. Consider using a different method to visually hide the element while keeping it accessible, such as `visually-hidden` class.